src/details/BookDetails.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import MarketplaceLink from './MarketplaceLink'
+import StarRating from './StarRating'
+import BookYearsReferenced from './BookYearsReferenced'
+
+class BookDetails extends Component {
+
+    createMarkup(book) {
+        return {__html: book.summary};
+    }
+
+    render() {
+        let book = this.props.book;
+        return (
+            <div className="row">
+                <div className="col-md-3 col-sm-12">
+                    <MarketplaceLink link={book.marketplaceLink}/>
+                </div>
+                <div className="col-md-9 col-sm-12">
+                    <h2 className="left-align">{book.title}</h2>
```

Comment:
using class names like this can get very confusing down the line. What if someday you change your style and you want these to be right aligned? It'll be easier to edit the css than to edit every name in your html code. 

I would come up with naming conventions for each page and each type of component. Eventually you'll have style sheets that only apply to a page at a time and this will save you a lot of headache.